index.js

```diff
@@ -30,32 +30,47 @@ module.exports = {
   treeForPublic(tree) {
     let trees = [];
     const targetBrand = this.options['@embroider/macros'].setOwnConfig.brand;
-    const pkgName = this.parent.pkg.name;
-    const isEngine = checkIfEngine(this.parent.pkg);
+    const ebmCandidates = this.parent.addons
```

Comment:
I don't get the meaning of the variable name - can you explain or be more explicit in the naming ?

Reply:
It was to addons that need to be targeted by ebm. Could also be something like `brandedTargets` ?